stub/src/main/java/io/grpc/stub/ServerCallStreamObserver.java

```diff
@@ -64,6 +64,18 @@ public abstract class ServerCallStreamObserver<RespT> extends CallStreamObserver
    */
   public abstract void setOnCancelHandler(Runnable onCancelHandler);
 
+
+  /**
+   * A hint to the call that specifies how many bytes must be queued before
+   * {@link #isReady()} will return true. A call may ignore this property if
+   * unsupported. This must be set before any messages are sent.
```

Comment:
The implementation is checking that the stream is still being initialized, which is different than "before any messages are sent." Say it has to be before both and keep the current verification? In `setOnCancelHandler` use the language "This method may only be called during the initial call to the application, before the service returns its {@code StreamObserver}."